metaci/build/models.py

```diff
@@ -194,6 +194,7 @@ def run(self):
         try:
             # Extract the repo to a temp build dir
             self.build_dir = self.checkout()
+            self.root_dir = os.getcwd()
```

Comment:
I think it'd be better to avoid storing root_dir on the build instance (since it's not a db field) and just pass root_dir to run() instead of the entire build instance.

Reply:
fixed in latest commit